tests/bofire/strategies/doe/test_utils.py

```diff
@@ -74,7 +74,7 @@ def test_get_formula_from_string():
     assert all(term in np.array(model_formula, dtype=str) for term in terms)
 
     # linear and quadratic
-    terms = ["1", "x0", "x1", "x2", "x0 ** 2", "x1 ** 2", "x2 ** 2"]
+    terms = ["1", "x0", "x1", "x2", "x0**2", "x1**2", "x2**2"]
```

Comment:
One can see it also here that your branch differes from main in `doe/test_utils.py`